app/models/episode.rb

```diff
@@ -32,4 +32,13 @@ def knocked_out
     end
   end
 
+  def to_json
+    {
+        name: name,
+        won_reward: 'Kyle',
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.